kijiji_repost_headless/kijiji_api.py

```diff
@@ -115,6 +116,11 @@ def login(self, username, password):
         }
         api_url = 'https://www.kijiji.ca/anvil/api'  # API endpoint
         resp = self.session.post(api_url, json=payload)
+        """
+        ssidDir = __file__[:-36] + "ssid.txt"
```

Comment:
Hard coding the number of characters in the current file path is a bad idea and not portable. This will break if the current file name or the parent directory name lengths are changed.
Use the `os.path` module to get the path to the parent directory.